notebooks/exploring_explorer.livemd

```diff
@@ -24,19 +24,17 @@ Mix.install([
 
 ## Reading and writing data
 
-Data can be read from delimited files (like CSV), Parquet, and the Arrow IPC (feather) format. You can also load in data from a map or keyword list of columns with `Explorer.DataFrame.new/1`.
+Data can be read from delimited files (like CSV), NDJSon, Parquet, and the Arrow IPC (feather) format. You can also load in data from a map or keyword list of columns with `Explorer.DataFrame.new/1`.
 
 For CSV, your 'usual suspects' of options are available:
 
 * `delimiter` - A single character used to separate fields within a record. (default: `","`)
-* `dtypes` - A keyword list of `[column_name: dtype]`. If `nil`, dtypes are imputed from the first 1000 rows. (default: `nil`)
-* `header?` - Does the file have a header of column names as the first row or not? (default: `true`)
-* `max_rows` - Maximum number of lines to read. (default: `Inf`)
-* `names` - A list of column names. Must match the width of the dataframe. (default: nil)
```

Comment:
This one is meant for renaming when you read it in or if it doesn't have a header. You could just pipe to `rename` I suppose. It's commonly supported in other languages but I don't think I've just about ever used it.

Reply:
Yeah. The issue is that it introduces ambiguity for the other option (the names before or after). We would also have to normalize options, etc. I think going with renaming is the cleaner way to do it. :) If necessary, we can add a mention to the docs about rename!